githttp.go

```diff
@@ -23,6 +31,8 @@ type GitHttp struct {
 
 	// Event handling functions
 	EventHandler func(ev Event)
+
+	PathResolver PathResolver
```

Comment:
golint would say:

```
warning: exported field PathResolver should have comment or be unexported
```

I think it's important to document that `PathResolver` is optional and can be `nil`. Otherwise, people who don't need to use this will be unsure what value to set here.